requirements.txt

```diff
@@ -1 +1,2 @@
 p4runtime==1.3.0
+Jinja2
```

Comment:
It is recommended to pin the version down for a repeatable/dependable build outcomes.

Some linters like Docker linters (which this is not), fail when versions are not pinned down